Source/Mosa.Platform.x64/Stages/FinalTweakStage.cs

```diff
@@ -94,7 +94,7 @@ public void MovLoad16(Context context)
 
 			Debug.Assert(result.IsCPURegister);
 
-			if (result.Register == CPURegister.ESI || result.Register == CPURegister.EDI)
+			if (result.Register == CPURegister.R6 || result.Register == CPURegister.R7)
```

Comment:
Should use `RSI` and `RDI` throughout the file so it's more obvious which special registers are being referred to.